specification/README.md

```diff
@@ -540,6 +540,8 @@ The tileset JSON has four top-level properties: `asset`, `properties`, `geometri
 
 `asset` is an object containing metadata about the entire tileset. The `asset.version` property is a string that defines the 3D Tiles version, which specifies the JSON schema for the tileset and the base set of tile formats.  The `tilesetVersion` property is an optional string that defines an application-specific version of a tileset, e.g., for when an existing tileset is updated.
 
+> **Implementation Note**: `tilesetVersion` can be used as a query parameter when requesting tiles to avoid using outdated content from a cache.
```

Comment:
```suggestion
> **Implementation Note**: `tilesetVersion` can be used as a query parameter when requesting content to avoid using outdated content from a cache.
```